ui/apps/everest/src/components/hidden-row/HiddenPasswordToggle.tsx

```diff
@@ -47,23 +47,33 @@ export const HiddenPasswordToggle = ({
         display: 'flex',
         flexDirection: 'row',
         alignItems: 'flex-start',
-        minHeight: '45px',
       }}
       data-testid="hidden-row"
     >
-      <Typography variant="body2" sx={{ wordBreak: 'break-all', pt: 0.5 }}>
+      <Typography
+        variant="body2"
+        sx={{ wordBreak: 'break-all', pt: show ? 0 : 0.5, pr: 1.5 }}
+      >
         {formattedValue}
       </Typography>
       <IconButton
         onClick={toggle}
+        size="xs"
         aria-label={`visibility-${show ? 'off' : 'on'}`}
-        sx={{ mt: -1 }}
+        sx={{ mt: -0.25 }}
       >
-        {show ? <VisibilityOutlinedIcon /> : <VisibilityOffOutlinedIcon />}
+        {
+          //TODO 1230 discussion with Nuna about common usage of this part
```

Comment:
![image](https://github.com/user-attachments/assets/407f9f17-a160-4d47-9458-95674a176a2c)
connection detailes section and overview part looks the same in new design, so Nuna agreed of using one component, 
![image](https://github.com/user-attachments/assets/99f99cb2-19f7-417e-bfb4-ff2a8987939e)
But in order not to delay this ticket, I suggest making this change directly in the task related to the dbDetails card => https://perconadev.atlassian.net/browse/EVEREST-1208